src/pages/Pool/CTACards.tsx

```diff
@@ -137,7 +144,7 @@ export default function CTACards() {
             </RowBetween>
           </ResponsiveColumn>
         </CTA1>
-        <CTA2 href={'https://info.uniswap.org/#/pools'}>
+        <CTA2 href={infoLink + '/pools'}>
```

Comment:
i don't love this method of constructing info urls because it implies you know what infoLink ends in when it's in another file, plus this is copy pasted 4 times that i count now (and the copy pasted code isn't particularly readable) so i would prefer to move this into a function

Reply:
Take a look at the new implementation and lmk what you think. It still uses the `infoLink + '/pools'` syntax in the templates, but it removes the tedious L2 checks from all the components.